sklearn/preprocessing/tests/test_label.py

```diff
@@ -44,6 +45,45 @@ def test_label_binarizer():
     assert_array_equal(expected, got)
     assert_array_equal(lb.inverse_transform(got), inp)
 
+    inp = [0, 0, 0, 0]
+    got = lb.fit_transform(inp)
+    assert_array_equal(lb.classes_, [0])
+    assert_array_equal(expected, got)
+    assert_array_equal(lb.inverse_transform(got), inp)
+
+    inp = [1, 1, 1, 1]
+    expected = np.array([[1, 1, 1, 1]]).T
+    got = lb.fit_transform(inp)
+    assert_array_equal(lb.classes_, [1])
+    assert_array_equal(expected, got)
+    assert_array_equal(lb.inverse_transform(got), inp)
+
+    # For sparse case:
+    lb = LabelBinarizer(sparse_output=True)
+    inp = ["pos", "pos", "pos", "pos"]
+    expected = np.array([[0, 0, 0, 0]]).T
```

Comment:
I just did that to make it symmetric with the `dense` case in line 41 which was already written.